optimum/exporters/onnx/model_configs.py

```diff
@@ -400,12 +400,14 @@ def _create_dummy_input_generator_classes(self, **kwargs) -> List["DummyInputGen
             self.task, self._normalized_config, **kwargs
         )
 
+        """
```

Comment:
Because of https://github.com/huggingface/transformers/commit/97a51b0c7d483cdf13ea878a987f9aa1c9eecc91#diff-ebaff1224cad711fd3cefb771ce17d1392ae2dfc7f74dc7da9dd014d7642a344

The `encoder_outputs` first item must use the full sequence length

Reply:
Does this statement `The encoder_outputs first item must use the full sequence length` holds true for all models. For encoders which are not text for example `Whisper`, the `encoder_hidden_states` is `[batch, feature_size / 2, hidden_size]`, here feature size is not equivalent to sequence_length. 

P.S. The whisper is failing in optimum also because of the above reason.  The `cross-attention` key values are not exported because of the conditional statement. I have pinged Sourab for more clarity on the following.

Reply:
Will look into it, thanks!